AirCasting/Map/GoogleMapView.swift

```diff
@@ -60,43 +60,69 @@ struct GoogleMapView: UIViewRepresentable {
             let coordinate = point.location
             path.add(coordinate)
         }
-        let polyline = GMSPolyline(path: path)
-        
+        let polyline = context.coordinator.polyline
         let spans = pathPoints.map { point -> GMSStyleSpan in
             let color = colorPolyline(point: point)
             return GMSStyleSpan(style: GMSStrokeStyle.solidColor(color),
                                 segments: 1)
         }
+
+        polyline.path = path
+        polyline.spans = spans
+        polyline.strokeWidth = 10
+        polyline.map = uiView
         
-        let updatedCameraPosition = setStartingPoint(points: pathPoints)
-        DispatchQueue.main.async {
-            uiView.camera = updatedCameraPosition
-        }
-        // Update starting point
-        if !context.coordinator.didSetInitLocation && !pathPoints.isEmpty {
+        // Update camera's starting point
+        if context.coordinator.shouldAutoTrack {
             let updatedCameraPosition = setStartingPoint(points: pathPoints)
             DispatchQueue.main.async {
-                uiView.camera = updatedCameraPosition
+                uiView.moveCamera(cameraUpdate)
             }
-            context.coordinator.didSetInitLocation = true
         }
-        polyline.spans = spans
-        polyline.strokeWidth = 6
-        polyline.map = uiView
+    }
+    
+    var cameraUpdate: GMSCameraUpdate {
+        guard !pathPoints.isEmpty else {
+            let location = tracker.googleLocation.last?.location ?? CLLocationCoordinate2D(latitude: 0, longitude: 0)
```

Comment:
why CLLocationCoordinate2D(latitude: 0, longitude: 0)? I think we have FakeLocation somewhere

Reply:
Yes, we do in CreateSessionDetailsView it's not global though.